src/__swaps__/screens/Swap/providers/swap-provider.tsx

```diff
@@ -606,6 +608,7 @@ export const SwapProvider = ({ children }: SwapProviderProps) => {
   }, []);
 
   const hasEnoughFundsForGas = useSharedValue<boolean | undefined>(undefined);
+  const gasAssetSymbol = useSharedValue<string>('Gas');
```

Comment:
i think we probably want this to initially be based of the `internalSelectedInputAsset.value`, right?

Reply:
I also think we could just name this `nativeAssetForChainSymbol`